1.MLP_Binary_Classifier/MLP_utils/utils.py

```diff
@@ -0,0 +1,881 @@
+"""
+Functions for Machine Learning Model optimization, training and testing.
+These are helper functions meant to be called in a separate notebook or script
+"""
+
+
+import ast
+from configparser import ConfigParser
+from typing import Tuple
+
+import matplotlib.pyplot as plt
+import numpy as np
+import optuna
+import pandas as pd
+import seaborn as sns
+import torch
+import torch.nn as nn
+import torch.optim as optim
+from MLP_utils.parameters import Parameters
+from sklearn.metrics import (
+    accuracy_score,
+    auc,
+    classification_report,
+    confusion_matrix,
+    f1_score,
+    precision_score,
+    recall_score,
+    roc_auc_score,
+    roc_curve,
+)
+from sklearn.model_selection import train_test_split
+
+config = ConfigParser()
+config.optionxform = str
+config.read("MLP_utils/config.ini")
+
+config["DEFAULT"]
+
+params = Parameters()
```

Comment:
Maybe too much to perform for now, but why not have this work:

```python
params = Parameters(config=config["DEFAULT"])
```

It seems like duplicating maintenance work to add methods to both the `Parameters` class, the `parameter_set` method, and the `config.ini`.

I also think it would be relatively straightforward to update `Parameters` as well. (Unless I'm not considering requirements for the `@dataclass` decorator.

Reply:
Erik and I actually were talking about this the other day and I think we might have a similar solution. I will add this to issues for a future PR!